src/http/common/entities/curated-operator.ts

```diff
@@ -1,34 +0,0 @@
-import { ApiProperty } from '@nestjs/swagger';
-import { Operator } from './';
-import { RegistryOperator } from '../../../common/registry';
-
-// TODO: it enough use operator from registry , lets remove
-export class CuratedOperator extends Operator {
```

Comment:
remains the mention of `CuratedOperator` in the readme

Reply:
It fixed in readme pr